tests/common/aws_test_runner.c

```diff
@@ -203,8 +166,6 @@ static void RunTestSuites( void )
         RUN_TEST_GROUP( BLE_Unit_MQTT_Serialize );
```

Comment:
is BLE_Unit_MQTT_Serialize test still needed?

Reply:
BLE has its own MQTT serializer in `libraries/ble/include/iot_ble_mqtt_serialize.h`